src/backend/distributed/citus.sql

```diff
@@ -194,6 +194,13 @@ COMMENT ON FUNCTION master_create_distributed_table(table_name regclass,
                                                     distribution_method citus.distribution_type)
     IS 'define the table distribution functions';
 
+CREATE FUNCTION master_update_shard_statistics(relation_id regclass, shard_id bigint)
```

Comment:
> This change probably requires an upgrade script.
CREATE FUNCTION master_update_shard_statistics(relation_id regclass, shard_id bigint)

Should it be enough to move this to citus--5.0--5.0-1.sql? I also suspect that we will rename it to citus--5.0--5.1.sql before the code-freeze?